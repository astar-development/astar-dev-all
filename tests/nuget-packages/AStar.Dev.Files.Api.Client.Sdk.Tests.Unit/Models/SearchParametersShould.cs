using AStar.Dev.Files.Api.Client.SDK.Models;

namespace AStar.Dev.Files.Api.Client.Sdk.Models;

public sealed class SearchParametersShould
{
    [Fact]
    public void ReturnTheExpectedToString() =>
        new SearchParameters().ToString().ShouldMatchApproved();

    [Fact]
    public void ReturnTheExpectedToQueryString() =>
        new SearchParameters().ToQueryString().ShouldMatchApproved();
}
