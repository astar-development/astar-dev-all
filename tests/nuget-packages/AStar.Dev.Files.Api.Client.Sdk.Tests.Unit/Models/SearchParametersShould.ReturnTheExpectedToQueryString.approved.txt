SearchFolder=&CurrentPage=1&ItemsPerPage=10&SearchType=Images&Recursive=True&IncludeSoftDeleted=False&IncludeMarkedForDeletion=False&ExcludeViewed=True&SortOrder=SizeDescending&MaximumSizeOfThumbnail=850&MaximumSizeOfImage=1500&ExcludeViewedPeriodInDays=7&SearchText=